### PR TITLE
Make LI period fact and LI Overview Dashboard

### PR DESCRIPTION
**Pull request by AnnaSerova**

### CODE REVIEW

File: li_period_fact.view.lkml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    datagroup_trigger: linkedin_ads_etl_datagroup
+    explore_source: linkedin_ads_ad_impressions {
+      column: _date { field: fact.date_date }
+      column: account_name { field: fact.account_name }
+      column: account_id { field: fact.account_id }
+      column: campaign_id { field: fact.campaign_id }
+      column: campaign_name { field: fact.campaign_name }
+      column: clicks { field: fact.total_clicks }
+      column: conversions { field: fact.total_conversions }
+      #column: conversionvalue { field: fact.conversion_value_in_local_currency }
```
Comment (jdavidson) [inline li_period_fact.view.lkml]: why is this commented out?

File: campaign_stats.view.lkml [hunk 2]
```diff
@@ -0,0 +1,241 @@
+include: "li_ad_metrics_base.view"
```
Comment (jdavidson) [inline campaign_stats.view.lkml]: I think this file is already present in the adapter

**Comment by AnnaSerova** [comment]
@jdavidson okay, fixed the issues you pointed out :)

**Comment by jdavidson** [comment]
you rock! thanks!

**Pull request merged**